sql/core/src/test/scala/org/apache/spark/sql/DatasetSuite.scala

```diff
@@ -2240,7 +2240,7 @@ class DatasetSuite extends QueryTest
       sqlState = None,
       parameters = Map(
         "objectName" -> "`nonexisting`",
-        "proposal" -> "`map`, `other.column`"))
+        "proposal" -> "`map`, `other`.`column`"))
```

Comment:
this is a bug, the column name is indeed `other.column`

Reply:
Yep, it is.